pkg/pillar/dpcmanager/dpcmanager_test.go

```diff
@@ -1134,28 +1104,9 @@ func TestWireless(test *testing.T) {
 	}).Should(BeTrue())
 
 	// Simulate some output from wwan microservice.
```

Comment:
Is this comment still valid? It seems now you are just updating the status and not publishing metrics and location anymore...

Reply:
Good point. Now the comment is properly updated.